index.js

```diff
@@ -292,7 +291,8 @@ function downloadYoutubeVideo(name, location = './', { quality = 'highest', logg
   return highland(videoSearchPromise) // search the video
     .flatMap(video => highland((push, next) => {
       if (!video) {
-        push(new Error("Video not found"));
+        warn(logger, `Video not found!`);
+        push(null, highland.nil);
```

Comment:
You can remove this call, as `throughStream` seems to do that already.

Reply:
Sorry about the branch, didn't know! =/

About this call, that was my first approach!
But without that, the process stops on the first _"Video not found"_ occurrence not allowing the playlist to continue fetching the next tracks.

As a user, I prefer to keep fetching all tracks even some is not found and that looks the desire in the issue too.

Reply:
Got it, I agree. Did you test downloading audio? Because it uses the `throughStream` approach:

https://github.com/danguilherme/spotivy/blob/5b5a75b47cfab852e8d6d61f7a38e2ce58e35153/index.js#L342-L345

Reply:
Hum... no, couldn't reproduce the scenário with audio!
But probably will be the same, I'll change that